templates/artist/artist_detail.html

```diff
@@ -152,6 +129,9 @@ <h1 class="header-font">{{ artist.name }}</h1>
                     .done(function(resp) {
                         // Show success
                         $('#success-modal > h1').text('SUCCESS!');
+                        $('#success-modal button.button').click(function() {
```

Comment:
I haven't been able to test this section because I can't get the invest module to work. I think there is something happening in the JS that is causing it to break